Display any text/* part with the internal viewer, see upstream bug
http://bugs.mutt.org/3246

--- a/muttlib.c
+++ b/muttlib.c
@@ -622,11 +622,8 @@
   switch (m->type)
   {
     case TYPETEXT:
-
-      if (!ascii_strcasecmp ("plain", m->subtype) ||
-	  !ascii_strcasecmp ("rfc822-headers", m->subtype) ||
-	  !ascii_strcasecmp ("enriched", m->subtype))
-	return 0;
+      /* we can display any text, overridable by auto_view */
+      return 0;
       break;
 
     case TYPEAPPLICATION:
